package.json

```diff
@@ -52,6 +52,7 @@
     "webpack": "^1.8.3"
   },
   "dependencies": {
+    "moment": "^2.10.2",
```

Comment:
Might be a good topic to add to the list for Friday's discussion. Can we for now, and our sanity, put everything in `devDependencies` to match fabricator's style? It's already violating your explanation as it stands, so we might as well start with 'em in one place? Argue back vociferously if your opinion here is strong. Mine is medium.